sdks/python/apache_beam/io/gcp/bigquery_read_it_test.py

```diff
@@ -674,6 +676,7 @@ def create_table(cls, table_name):
 
     cls.bigquery_client.insert_rows(
         cls.project, cls.dataset_id, table_name, table_data)
+    mod_time.sleep(10)
```

Comment:
if we have to wait, can we move this 10 seconds to a global variable?

Reply:
Replaced to `get_table` method